ci-operator/jobs/che-incubator/chectl/OWNERS

```diff
@@ -1,9 +1,5 @@
 approvers:
-- rhopp
-- flacatus
-- tolusha
+- abazko
```

Comment:
My github nick name is `tolusha`, could you also add `svor` to the lists?

Reply:
Fixed